app/scripts/models/account.js

```diff
@@ -474,6 +474,7 @@ define(function (require, exports, module) {
         }
 
         this.set(updatedSessionData);
+        this._notifier.trigger('set-uid', this.get('uid'));
```

Comment:
I understand the desire to decouple modules using events, but this module already contains a reference to the metrics module so that `this._metrics.getFlowEventMetadata()` can be invoked. Does using an imperative command here offer anything that invoking `this._metrics.setUid` directly does not? If so, maybe a non-imperative name, something like `signed-in`/`signed-out`? Total bike-shed territory here.